stack-8.0.1.yaml

```diff
@@ -0,0 +1,14 @@
+resolver: lts-7.24
```

Comment:
@power-fungus could we add these `yaml` files under `stack/` directory? similar to what i've done with `nix`?

Reply:
Sounds fine.